common/src/main/java/com/revenuecat/purchases/common/BillingWrapper.kt

```diff
@@ -134,23 +138,26 @@ class BillingWrapper(
         onReceiveSkuDetails: (List<SkuDetails>) -> Unit,
         onError: (PurchasesError) -> Unit
     ) {
-        debugLog("Requesting products with identifiers: ${skuList.joinToString()}")
+        log(LogIntent.DEBUG_INFO, OfferingStrings.FETCHING_PRODUCTS.format(skuList.joinToString()))
         executeRequestOnUIThread { connectionError ->
             if (connectionError == null) {
                 val params = SkuDetailsParams.newBuilder()
                     .setType(itemType).setSkusList(skuList).build()
                 withConnectedClient {
                     querySkuDetailsAsync(params) { billingResult, skuDetailsList ->
                         if (billingResult.responseCode == BillingClient.BillingResponseCode.OK) {
-                            debugLog("Products request finished for ${skuList.joinToString()}")
-                            debugLog("Retrieved skuDetailsList: ${skuDetailsList?.joinToString { it.toString() }}")
+                            log(LogIntent.DEBUG_INFO, OfferingStrings.FINISH_FETCHING_PRODUCTS
+                                    .format(skuList.joinToString()))
+                            log(LogIntent.PURCHASE, OfferingStrings.RETRIEVED_SKU
+                                    .format(skuDetailsList?.joinToString { it.toString() }))
                             skuDetailsList?.takeUnless { it.isEmpty() }?.forEach {
-                                debugLog("${it.sku} - $it")
+                                log(LogIntent.PURCHASE, OfferingStrings.PRODUCTS.format(it.sku, it))
                             }
 
                             onReceiveSkuDetails(skuDetailsList ?: emptyList())
                         } else {
-                            infoLog("Error when fetching products. ${billingResult.toHumanReadableDescription()}")
+                            log(LogIntent.GOOGLE_ERROR, OfferingStrings.INVALID_PRODUCT_IDENTIFIERS
+                                    .format(billingResult.toHumanReadableDescription()))
```

Comment:
this seems like a bit of a jump - how did we go from "error when fetching products" to invalid product identifiers?